client/src/lit/interactive-example/global.scss

```diff
@@ -2,9 +2,13 @@
 
 interactive-example {
   display: block;
-  height: 444px;
+  height: 375px;
   margin: 1rem 0;
 
+  &[height="js"] {
+    height: 444px;
+  }
```

Comment:
This doesn't work, there's no `height=js` attribute set anywhere

Reply:
Removed in 0090e2932f.